modules/benchmark/R/load.data.R

```diff
@@ -16,6 +16,7 @@ load.data <- function(data.path, format, start_year = NA, end_year=NA, site=NA,
   require(PEcAn.benchmark)
   require(lubridate)
   require(udunits2)
+  require(dplyr)
```

Comment:
CRAN guidance:
>If a package only needs a few objects from another package it can use a fully qualified variable reference in the code instead of a formal import. A fully qualified reference to the function f in package foo is of the form foo::f. This is slightly less efficient than a formal import and also loses the advantage of recording all dependencies in the NAMESPACE file (but they still need to be recorded in the DESCRIPTION file). Evaluating foo::f will cause package foo to be loaded, but not attached, if it was not loaded already—this can be an advantage in delaying the loading of a rarely used package.

This is basically what I do. If it's a one-off, OK, use `::`, but if much more than that, do it via the `NAMESPACE` file.